### ISSUE TITLE
Travis build fails due to undocumented code objects

### ISSUE THREAD

**Issue by richelbilderbeek** [rsetienne/DAISIE#7]
[The Travis CI build log](https://travis-ci.org/rsetienne/DAISIE/builds/499374314#L4397) states:

```
Undocumented code objects:
  ‘DAISIE_loglik_all_choosepar3’ ‘DAISIE_ML3’ ‘order_pars1’
  ‘translate_island_ontogeny’
```

Apparently, there is some undocumented code that breaks the build.

I can only suggest the authors of those functions to document these.

**Comment by richelbilderbeek** [comment]
@Neves-P: I'll assign you, so that you can sort this out with the team. Feel free to unassign, assign me, etc.

**Comment by richelbilderbeek** [comment]
P.S. One simple solution: convert those functions to Roxygen and it should work :+1:

**Comment by rsetienne** [comment]
Did that.

**Comment by Neves-P** [comment]
Fixed in ff009d4f052c3f97d47a70c2698bc08e37d61721, thanks both!
